content/en/docs/tutorials/online-training/overview.md

```diff
@@ -25,7 +25,9 @@ Here are some of the sites that offer online training for Kubernetes:
 
 * [Kubernetes the Hard Way (LinuxAcademy.com)](https://linuxacademy.com/linux/training/course/name/kubernetes-the-hard-way)
 
-* [Certified Kubernetes Application Developer Preparation Course (KodeKloud.com)](https://kodekloud.com/p/kubernetes-certification-course)
+* [Kubernetes for the Absolute Beginners with Hands-on Labs (KodeKloud.com)](https://kodekloud.com/p/kubernetes-for-the-absolute-beginners-hands-on?coupon_code=KUBEIO)
```

Comment:
What is the purpose of the coupon_code?

Reply:
It gives a special discount to students while enrolling in the course.  Can be removed if not appropriate.